pingpong/test_lecture_video_server.py

```
@@ -3491,6 +5066,312 @@ async def test_get_thread_video_invalid_range_returns_416(
 
 
+@with_user(123)
+@with_institution(11, "Test Institution")
+@with_authz(
+    grants=[
+        ("user:123", "can_create_thread", "class:1"),
+        ("user:123", "student", "class:1"),
+    ]
+)
+async def test_get_thread_lecture_video_narration_streams_audio(
+    api, authz, db, institution, valid_user_token, config, monkeypatch, tmp_path
```

Ruff (0.15.5)

[warning] Unused function argument: `authz`

(ARG001)